packages/ui/src/theme/defaultDarkModeOverride.ts

```diff
@@ -104,6 +104,12 @@ const darkModeTokens = {
       tertiary: '{colors.neutral.20}',
     },
 
+    shadow: {
+      primary: { value: 'hsla(100, 100%, 100%, 0.25)' },
+      secondary: { value: 'hsla(100, 100%, 100%, 0.15)' },
+      tertiary: { value: 'hsla(100, 100%, 100%, 0.05)' },
```

Comment:
Nit: Is this tertiary value being used anywhere besides the slider? The alpha of 0.05 makes sense to match the light mode counterpart, but it's so subtle in dark mode that I can't even see it. Would it make sense to consider increasing the tertiary alpha value for dark mode?